### PR TITLE
[18.0][MIG] l10n_fi_edicode

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**

### CODE REVIEW

**Comment by JordiMForgeFlow** [comment]
@OCA/local-finland-maintainers could we merge this one? :)

**Comment by jarmokortetjarvi** [comment]
Looks good :+1: 

Diff with `17.0`
https://github.com/OCA/l10n-finland/compare/17.0...ForgeFlow:l10n-finland:18.0-mig-l10n_fi_edicode

File: l10n_fi_edicode/models/res_partner_operator_einvoice.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    @api.depends("identifier")
+    def _compute_display_name(self):
+        """
+        Overwrite core method to add value of `identifier` ("Identifier") field
+        into name of records.
+        """
+        res = super()._compute_display_name()
+        for operator in self:
+            operator.display_name = " - ".join([operator.identifier, operator.name])
+        return res
```
Comment (mlaitinen) [inline l10n_fi_edicode/models/res_partner_operator_einvoice.py:44]: What's the purpose of returning `res` in a compute method?

Comment (JordiMForgeFlow) [reply l10n_fi_edicode/models/res_partner_operator_einvoice.py:44]: it is a pylint check (W8110) that makes sure that you do not break the result of calling 'super'. Obviously in the compute methods you should not expect any result, but it is done for convenience and good practice

https://github.com/OCA/pylint-odoo/blob/6e857c52117ef8597e0433b32ca6fac302ea76b8/src/pylint_odoo/checkers/odoo_addons.py#L208

Comment (mlaitinen) [reply l10n_fi_edicode/models/res_partner_operator_einvoice.py:44]: Welp, I don't see how it could cause any harm.

**Review by mlaitinen — APPROVED** [review]

**Pull request merged**